packages/standard/zarf.yaml

```diff
@@ -99,7 +99,7 @@ components:
 
   # UDS Runtime
   - name: uds-runtime
-    required: false
+    required: true
```

Comment:
Should be able to also drop from [the bundle](https://github.com/defenseunicorns/uds-core/blob/3c5d3416ffb3141f40b83601b94699e532a572f0/bundles/k3d-standard/uds-bundle.yaml#L45) now.

Reply:
done!